crux_db/crux.sh

```diff
@@ -263,115 +253,120 @@ echo " "
 echo "Part 1.3:"
 echo "Clean ${NAME} ecoPCR output for blasting and run BLAST jobs"
 
-## loop through ecoPCR results
```

Comment:
Ok I actually think instead of adding an argument to skip blast, we should just remove blast altogether and replace with Bowtie2. So you can go back to the old structure of this part of the code and remove the if (skipblast) statements, but just have it call bowtie2 instead